src/mds/MDSDaemon.cc

```diff
@@ -367,11 +367,11 @@ void MDSDaemon::set_up_admin_socket()
 				     asok_hook,
 				     "List client sessions based on a filter");
   ceph_assert(r == 0);
-  r = admin_socket->register_command("session evict name=filters,type=CephString,n=N,req=false",
+  r = admin_socket->register_command("session evict name=filters,type=CephString,n=N,req=true",
```

Comment:
What's the significance of `n=N` here ? From the first glimpse, it looks like if `n=N` is passed, `req` flag is not honored?

https://github.com/ceph/ceph/blob/main/src/common/cmdparse.cc#L664
https://github.com/ceph/ceph/blob/main/src/common/cmdparse.cc#L613